packages/mc-html-template/src/replace-html-placeholders.ts

```diff
@@ -1,22 +1,33 @@
-const htmlScripts = require('../load-html-scripts');
-const htmlStyles = require('../load-html-styles');
-const sanitizeAppEnvironment = require('./sanitize-app-environment');
+import type { ApplicationRuntimeConfig } from '@commercetools-frontend/application-config';
+import sanitizeAppEnvironment from './utils/sanitize-app-environment';
+// https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros
+import htmlScripts from /* preval */ './load-html-scripts';
+// https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros
+import htmlStyles from /* preval */ './load-html-styles';
 
-const trimTrailingSlash = (value) => value.replace(/\/$/, '');
+type TReplaceHtmlPlaceholdersOptions = {
+  env: ApplicationRuntimeConfig['env'];
+  headers: Record<string, string | undefined>;
+};
```

Comment:
Previously we supported another prop called `cliFlags` I guess we remove because it wasn't actually used.
Perhaps we need to update the README since it was mentioned [over there](https://github.com/commercetools/merchant-center-application-kit/blob/ef163df49d3ba04f05e2b745a5b8b7f7203a1dc5/packages/mc-html-template/README.md?plain=1#L23).

Reply:
Good point about the README, I'll update that.